curvlinops/_base.py

```diff
@@ -324,7 +325,11 @@ def _postprocess(self, M_list: List[Tensor]) -> ndarray:
             concatenated dimensions over all list entries.
         """
         result = [rearrange(M, "k ... -> (...) k") for M in M_list]
-        return cat(result, dim=0).cpu().numpy()
+        result = cat(result)
+        # calling .numpy() on a BF-16 tensor is not supported, see
+        # (https://github.com/pytorch/pytorch/issues/90574)
+        result = result.float() if result.dtype == bfloat16 else result
+        return result.cpu().numpy().astype(self.dtype)
```

Comment:
Will `self.dtype` ever be anything else but `float32`? It seems it is hard-coded [here](https://github.com/f-dangel/curvlinops/blob/main/curvlinops/_base.py#L102), but not sure if this is overridden somewhere else.

Reply:
It is hard-coded. A better way to resolve this would be to expose the `dtype` in the constructor of all linear operators, but I'm hesitant to do this as I'm planning to deprecate the base class soon.

Reply:
I think we can ignore for now that the dtype issue does not present itself for the state-less linear operators.